main_test.go

```diff
@@ -713,3 +716,37 @@ func Test_runScorecardSettings(t *testing.T) {
 		})
 	}
 }
+
+func Test_signScorecardResult(t *testing.T) {
+	t.Parallel()
+	// Sign example scorecard results file.
+	scorecardResultsFile := "./testdata/scorecard-results-example.sarif"
+	err := signScorecardResult(scorecardResultsFile)
+	if err != nil {
+		t.Errorf("signScorecardResult() error, %v", err)
+		return
+	}
+
+	// Verify that the signature was created and uploaded to the Rekor tlog by looking up the payload.
+	ctx := context.Background()
+	rekorClient, err := rekor.NewClient("https://rekor.sigstore.dev")
+	if err != nil {
+		t.Errorf("signScorecardResult() error getting Rekor client, %v", err)
+		return
+	}
+	scorecardResultData, err := ioutil.ReadFile("./testdata/scorecard-results-example.sarif")
```

Comment:
`./testdata/scorecard-results-example.sarif` is used twice, let's create a variable for it